bot/exts/utilities/bookmark.py

```diff
@@ -152,6 +204,27 @@ def event_check(reaction: discord.Reaction, user: discord.Member) -> bool:
 
         await reaction_message.delete()
 
+    @commands.dm_only()
+    @bookmark.command(name="delete", aliases=("del", "rm"), root_aliases=("unbm", "unbookmark"))
+    async def delete_bookmark(
+        self,
+        ctx: commands.Context,
+        message_to_delete: Optional[WrappedMessageConverter]
+    ) -> None:
+        """
+        Delete the referenced DM message by the member.
+
+        Members can either give a message as an argument, or reply to a message.
+        """
+        message_to_delete = message_to_delete or getattr(ctx.message.reference, "resolved", None)
+        if not message_to_delete:
+            raise commands.UserInputError(MESSAGE_NOT_FOUND_ERROR)
+
+        if message_to_delete.channel != ctx.channel:
+            raise commands.UserInputError(":x: You can only delete messages in your own DMs!")
+        await message_to_delete.delete()
```

Comment:
*technically* this could be hacked through. 

As far as I can tell, the embed title has been "Bookmark" for all bookmarks sent in dms.

Reply:
imo it would not be bad if it allows other messages to be deleted as well. Also there wouldn't be need for storing bookmarks anywhere in that case. The original idea was for bookmarks but this does not sound bad idea if it allows to delete other messages.

Reply:
Yeah it isn't a bad idea, but IMO it's a bit of a hack and weird to make sir-lancebot delete any of its messages using a command called `.unbm`

Reply:
Ye, the `.unbm` doesn't fit there in that case. If it would be allowed to delete any messages the `.unbm` would be something like `.del`/`.delete` that should be existing.

Reply:
I chose not to limit this to only message IDs in redis as then users wouldn't be able to delete messages bookmarked before this PR was merged.

I think root-aliasing this as .delete could be good, especially with the customer dm check I will add which will explain the interface if ran in-server.